### ISSUE TITLE
Bug Focus ring is cut off in mega menu navigation

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#5997]
## Description
The focus ring is getting cut off when tabbing through nav items in the mega menu. See screenshot. @jeana-adhoc found this during testing of an issue with the megamenu and discovered that this bug has been around for awhile. Suggestion from Jeana: 
- if we change mm-link-container to be width:13.625rem things fit.

## Details
[Slack thread](https://dsva.slack.com/archives/C08MQHWV6VD/p1774371264108759)
<img width="1948" height="1564" alt="Image" src="https://github.com/user-attachments/assets/7bdf1699-9a43-4b47-9353-ed0fefca1794" />

## Tasks
- [ ] Fix focus ring so that it doesn't get cut off
- [ ] Have accessibility specialist review and test
- [ ] Publish fix

## Acceptance Criteria
- [ ] Tasks above are completed
